.github/workflows/ci.yaml

```diff
@@ -56,8 +59,19 @@ jobs:
         path: QuantLib-Risks-Cpp
     - name: Setup
       run: |
-        sudo apt update
-        sudo apt install -y libboost-dev ccache ninja-build
+        sudo apt-get update \
```

Comment:
I think we can avoid all of this if we simply use the same build container that QuantLib uses directly. Using the `container` argument - as it's done [here](https://github.com/lballabio/QuantLib/blob/master/.github/workflows/linux.yml#L136) with the `rolling` tag.  The way we follow the QuantLib pre-requisits and we don't need to install all these dependencies every time.